packages/fractor-htaccess/tests/Fixtures/RemoveSetEnvIfRule.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\FractorHtaccess\Tests\Fixtures;
+
+use a9f\FractorHtaccess\Contract\HtaccessFractorRule;
+use Symfony\Component\DependencyInjection\Exception\BadMethodCallException;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+use Tivie\HtaccessParser\HtaccessContainer;
+use Tivie\HtaccessParser\Token\Block;
+use Tivie\HtaccessParser\Token\TokenInterface;
+use const Tivie\HtaccessParser\Token\TOKEN_DIRECTIVE;
+
+final class RemoveSetEnvIfRule implements HtaccessFractorRule
```

Comment:
No idea why that rule might be necessary; would probably be good to have a short comment here.

Reply:
that is a test rule which won't be in the release package